src/opnsense/service/templates/OPNsense/Unbound/core/blocklists.conf

```diff
@@ -51,6 +51,12 @@ custom_{{loop.index}}={{uri}}
 {%      endfor %}
 {%    endif %}
 
+{%    if not helpers.empty('OPNsense.unboundplus.dnsbl.localzone')%}
```

Comment:
When we require both fields this conditional can be removed.

Reply:
Removed in next push